cmd/drone-server/server.go

```diff
@@ -157,28 +158,32 @@ var flags = []cli.Flag{
 	// resource limit parameters
 	//
 	cli.Int64Flag{
-		Name:   "limit-mem-swap",
 		EnvVar: "DRONE_LIMIT_MEM_SWAP",
+		Name:   "limit-mem-swap",
+		Usage:  "maximum swappable memory allowed in bytes",
 	},
 	cli.Int64Flag{
-		Name:   "limit-mem",
 		EnvVar: "DRONE_LIMIT_MEM",
+		Name:   "limit-mem",
+		Usage:  "maximum memory allowed in bytes",
 	},
 	cli.Int64Flag{
-		Name:   "limit-shm-size",
 		EnvVar: "DRONE_LIMIT_SHM_SIZE",
+		Name:   "limit-shm-size",
```

Comment:
No usage here, couldn't find any docs about this on https://docs.docker.com/engine/admin/resource_constraints/.

Reply:
https://docs.docker.com/compose/compose-file/#shm_size

Reply:
Added